config/routes.rb

```diff
@@ -25,6 +25,8 @@
   
   resources :posts
 
+  get 'live_feed/:id' => 'posts#around'
```

Comment:
The naming of the routes should be more consistent with the method names.

For example, if the url is live_feed, then you should have a `live_feed` method for that route, but currently it's a helper function.